config/core/build-configs-chromeos.yaml

```diff
@@ -46,6 +46,24 @@ chromeos_variants: &chromeos-variants
         filters:
           - passlist: { defconfig: ['x86-chromebook'] }
 
+chromeos_nocompress_variants: &chromeos-nocompress-variants
+  chromeos-clang-14:
+    build_environment: clang-14
+    architectures:
+      arm: *chromeos_variants_arm
+      arm64: *chromeos_variants_arm64
+      x86_64:
```

Comment:
I think you can also use this syntax to avoid having to add anchors for arm and arm64:
```yaml
chromeos_variants: &chromeos-variants
  chromeos-clang-14:
    build_environment: clang-14
    architectures: &chromeos-variants-architectures
    # ...
    
chromeos_nocompress_variants:
  chromeos-clang-14:
    build_environment: clang-14
    architectures:
      <<: *chromeos-variants-architectures
      x86_64:
        # variants here...
```

It's quite minor but should be a smaller change, and if we add architectures to the base variants they will be automatically inherited in the no-compress variants.

Reply:
Also if you use `<<: *chromeos-variants` you don't have to add a `clang-14` anchor using the same principle.

Reply:
Updated, i am curious now if our parser will process this properly, as some linters choked on it.
(Opened ticket for tool: https://sourceforge.net/p/ruamel-yaml-cmd/tickets/11/ )